mesheryctl/internal/cli/root/system/start.go

```diff
@@ -436,7 +436,7 @@ func start() error {
 		}
 
 		if err == nil {
-			err = utils.ChangeConfigEndpoint(mctlCfg.CurrentContext, currCtx)
```

Comment:
Instead of this new function, `ChangeConfigEndpoint`should be a function of the context struct and to change the endpoint, we first change the endpoint in the central struct and then write that central struct to the meshconfig file.